packages/manager/modules/octavia-load-balancer/src/create/service.js

```diff
@@ -0,0 +1,45 @@
+import { NETWORK_PRIVATE_VISIBILITY } from './constants';
+
+export default class OctaviaLoadBalancerCreateService {
+  /* @ngInject */
+  constructor($q, $http) {
+    this.$q = $q;
+    this.$http = $http;
+  }
+
+  getPrivateNetworks(projectId, regionName) {
+    return this.$http
+      .get(`/cloud/project/${projectId}/region/${regionName}/network`)
+      .then(({ data }) => {
+        const privateNetworks = data.filter(
+          (network) => network.visibility === NETWORK_PRIVATE_VISIBILITY,
+        );
+        return privateNetworks;
+      });
+  }
+
+  getSubnets(projectId, regionName, privateNetwork) {
+    return this.$http
+      .get(
+        `/cloud/project/${projectId}/region/${regionName}/network/${privateNetwork.id}/subnet`,
+      )
+      .then(({ data }) => {
+        const subnets = data.map((subnet) => ({
+          ...subnet,
+          displayName:
+            subnet.name !== ''
```

Comment:
What about if name == `null` or `undefined` ?

Reply:
It should not happen but in case of I can add the checks